packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -208,11 +208,11 @@ EEWorkflowController.get(
 			req.user,
 			req.query.filter,
 		)) as unknown as WorkflowEntity[];
+		await EEWorkflows.addCredentialsToWorkflows(workflows, req.user);
 
 		return Promise.all(
 			workflows.map(async (workflow) => {
 				EEWorkflows.addOwnerAndSharings(workflow);
```

Comment:
Can you please do the same for `addOwnerAndSharings` as well ?

Reply:
Considering this function does not touch the database, it simply massages data in place, it's safe to keep it in a `.map` - it is also used elsewhere so it's ok to stay like this.
I've removed the unnecessary `Promise.all` since there are no `async` calls